lib/configuration.sh

```diff
@@ -161,47 +161,70 @@ PACKAGE_LIST_DESKTOP="xserver-xorg xserver-xorg-video-fbdev gvfs-backends gvfs-f
 PACKAGE_LIST_DESKTOP_RECOMMENDS="mirage galculator hexchat xfce4-screenshooter network-manager-openvpn-gnome mpv fbi cups-pk-helper \
 	cups geany atril xarchiver"
 
+
+# For minimal build different set of packages is needed
+if [[ $BUILD_MINIMAL == yes  ]]; then
+
+	# Essential packages for minimal build
+	PACKAGE_LIST="bc cpufrequtils device-tree-compiler fping \
+		fake-hwclock psmisc ntp parted sudo linux-base dialog \
+		ncurses-term sysfsutils toilet u-boot-tools unattended-upgrades \
+		usbutils console-setup openssh-server initramfs-tools \
+		ca-certificates resolvconf iptables nocache debconf-utils"
```

Comment:
resolvconf and iptables can also be left out for systemd / network manager driven systems.

Reply:
Removed in caf67f8